pkg/authorizer/token_retriever_wi.go

```diff
@@ -0,0 +1,48 @@
+package authorizer
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/confidential"
+)
+
+// Get auth token from service account token
+func AcquireARMTokenFromServiceAccountToken(ctx context.Context, tenantID, clientID string) (types.AccessToken, error) {
```

Comment:
In `token_retriever.go`, they use a `TokenRetriever` object. Could we do something similar?